### PR TITLE
 #4141 new handler and route to retreive all systems

### PR DESCRIPTION
**Pull request by KaliGreen**

### CODE REVIEW

File: packages/user-mgnt/src/features/system/handler.ts [hunk 1]
```diff
@@ -321,6 +328,24 @@ export async function getSystemHandler(
...
+      system.name[0]?.family || ''
+    }`.trim()
+    allSystemsResponse.client_id = system.client_id
+    allSystemsResponse.name = systemName
+    allSystemsResponse.sha_secret = system.sha_secret
+    allSystemsResponse.status = system.status
+    allSystemsResponseArray.push(allSystemsResponse)
+  })
+  return allSystemsResponseArray
+}
```
Comment (naftis) [inline packages/user-mgnt/src/features/system/handler.ts:347]: Would a `.map` be more easy to use here:
```
export async function getAllSystemsHandler() {
  const systems = await System.find()

  return systems.map((system) => {
    const systemName = `${system.name[0]?.given || ''} ${
      system.name[0]?.family || ''
    }`.trim()

    return {
      client_id: system.client_id,
      name: systemName,
      sha_secret: system.sha_secret,
      status: system.status
    }
  })
}
```

Later, when we refactor the system name (it's an upcoming issue), we could even change this to something like:
```
export async function getAllSystemsHandler() {
  return System.find().select('client_id name sha_secret status');
}
```

File: packages/user-mgnt/src/config/routes.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ import {
   verifySystemResSchema,
   getSystemRequestSchema,
   getSystemResponseSchema,
-  getSystemHandler
+  getSystemHandler, getAllSystemsHandler
```
Comment (naftis) [inline packages/user-mgnt/src/config/routes.ts:64]: Run Prettier code formatting here.

**Review by naftis — CHANGES_REQUESTED** [review]
Hi, thanks for PR, small changes requested

File: packages/user-mgnt/src/features/system/handler.ts (hunk 1 above)
Comment (KaliGreen) [reply packages/user-mgnt/src/features/system/handler.ts:347]: Yeah that can work, looks shorter

**Comment by naftis** [comment]
lets pull `ocrvs-2277-integration-configuration` branch into this one, otherwise LGTM

**Pull request merged**